dashboard/app/models/census/census_summary.rb

```diff
@@ -284,24 +288,19 @@ def self.compute_teaches_cs(overrides_summary, has_ap_data, has_ib_data, submiss
     result = conditional_result('state_offering', state_summary, state_summary, state_value, result, details)
 
     result = conditional_result(
-      'consistent_non_teacher_surveys',
-      submissions_summary[:consistency][:not_teacher_or_admin],
-      submissions_summary[:consistency][:not_teacher_or_admin],
-      submissions_summary[:counts][:not_teacher_or_admin],
+      'inconsistent_teacher_surveys',
+      submissions_summary[:has_inconsistent_surveys][:teacher_or_admin],
+      'MAYBE',
+      submissions_summary[:counts][:teacher_or_admin],
       result,
       details
     )
```

Comment:
With this many parameters could it make sense to pass a hash of named arguments?  (Out of scope for this PR of course.)

Reply:
Yes. I'd like to refactor this whole flow. It requires too much effort to follow right now. I may try to do that in the next week or two if I can make the time.